apps/server/src/adapters/WebsocketAdapter.ts

```diff
@@ -24,14 +24,17 @@ import { IAdapter } from './IAdapter.js';
 import { eventStore } from '../stores/EventStore.js';
 import { logger } from '../classes/Logger.js';
 import { dispatchFromAdapter } from '../api-integration/integration.controller.js';
+import { generateId } from 'ontime-utils';
 
 let instance: SocketServer | null = null;
 
+type WsClient = Client & { ws: WebSocket };
```

Comment:
question: I am not sure why do we keep the websocket object here? is it not easier to keep just the id of the client. the websocket connection has its own list of clients and I dont think we want the two to become coupled

Reply:
to be able to send the rename from one ws client to another one if it should not be in the list object?

Reply:
I am not sure I understand, we can chat about it.
But we have a client ID, which is unique and doesnt change. As far as I can see, this ID is the only thing we need to share with the websocket object